src/server/gc_worker/gc_worker.rs

```diff
@@ -782,7 +783,7 @@ where
         cfg: GcConfig,
         feature_gate: FeatureGate,
     ) -> GcWorker<E, RR> {
-        let worker_builder = WorkerBuilder::new("gc-worker");
+        let worker_builder = WorkerBuilder::new("gc-worker").pending_capacity(GC_MAX_PENDING_TASKS);
```

Comment:
Is compaction filter the only source of the messages?

Reply:
for other messages like `GCTask::GC`, `GCTask::UnsafeDestroyRange` and `GCTask::PhysicalScanLock`, they are handled by `handle_gc_task_schedule_error` when encountering error.